src/Components/HOC/AuthHoc.tsx

```diff
@@ -14,6 +21,14 @@ const AuthHoc = ({ children, heading, subheading }) => {
           <p className={styles.subheading}>{subheading}</p>
         </div>
         <div className={styles.formWrapper}>{children}</div>
+        {stepperIsVisible && (
+          <div className={step === 3 ? `${styles.flexGap} ${styles.flex_margin}` : styles.flexGap}>
```

Comment:
using same styles for step = 1,2,3,4?

Reply:
done.

Reply:
yes using same styles, now used map instead of hardcoding.